chart/templates/_helpers.tpl

```diff
@@ -1286,7 +1290,11 @@ Return the directory where the proxy CA cert will be mounted
 Return the filename where the proxy CA will be mounted
 */}}
 {{- define "carto.proxy.configMapMountFilename" -}}
+{{- if .Values.externalProxy.sslCA -}}
 {{- print "ca.crt" -}}
+{{- else if .Values.externalProxy.sslCAConfigmapName -}}
+{{- print "ca_0.crt" -}}
```

Comment:
Confirming with Replicated team is that's the right approach here https://cartoteam.slack.com/archives/C05V5V84W7M/p1752499969534919